### PR TITLE
Changed min avilable birthday for user

### PR DESCRIPTION
**Pull request by Tali32**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by nicoolle — APPROVED** [review]

**Review by christykobyletska — APPROVED** [review]

File: src/pages/userPage/EditUserPage/EditUserPage.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default function () {
   const wrongOnlyLettersMessage = shouldContain("тільки літери");
   const wrongAllVariantsMessage = shouldContain("літери, символи та цифри");
   const [form] = Form.useForm();
-  const MAX_AGE = 100;
+  const MIN_AVILABLE_DATE = "01.01.1900";
```
Comment (ostappppppp) [inline src/pages/userPage/EditUserPage/EditUserPage.tsx:54]: I guess it should be:
```suggestion
  const MIN_AVAILABLE_DATE = "01.01.1900";
```

**Review by ostappppppp — CHANGES_REQUESTED** [review]

**Review by OleksandrBilan — APPROVED** [review]

File: src/pages/userPage/EditUserPage/EditUserPage.tsx (hunk 1 above)
Comment (Tali32) [reply src/pages/userPage/EditUserPage/EditUserPage.tsx:54]: Thanks)

Comment (Tali32) [reply src/pages/userPage/EditUserPage/EditUserPage.tsx]: Done!

**Review by Tali32 — COMMENTED** [review]
Done

**Review by ostappppppp — APPROVED** [review]

**Pull request merged**